src/api/app/models/event_subscription/generate_hash_for_subscriber.rb

```diff
@@ -0,0 +1,64 @@
+class EventSubscription
```

Comment:
This is not business logic, this querying data into a specific structure that a specific view uses right?